pkg/executor/plugins/rpc.go

```diff
@@ -0,0 +1,177 @@
+package plugins
+
+import (
+	"context"
+	"io"
+	"net/rpc"
+
+	"github.com/bacalhau-project/bacalhau/pkg/bidstrategy"
+	"github.com/bacalhau-project/bacalhau/pkg/model"
+)
+
+type ServerRPC struct {
```

Comment:
Server that translates RPC calls to arguments passed to the executor and translates responses from the executor to RPC.